packages/aggregator/src/http/server.rs

```diff
@@ -24,13 +31,14 @@ const REALM: &str = "aggregator";
 pub fn start(
     ctx: AppContext,
     config: Config,
+    chain_configs: Arc<RwLock<ChainConfigs>>,
```

Comment:
mentioned below but I suppose it's a bit easier to surface here - would be great if we could just make this part of `Config` itself